src/components/LeftNav.tsx

```diff
@@ -400,8 +405,8 @@ function Heading({
         to={path}
         style={headingStyle as StyleHTMLAttributes<HTMLElement>}
         className="flex"
-        activeStyle={activeStyle}
-        exact
+        activestyle={activestyle}
+        end
```

Comment:
What is this end attribute do? haven't seen it before

Reply:
https://v5.reactrouter.com/web/api/NavLink/activestyle-object it was a  NavLink attribute in RR5, but they got rid of it. How ever they got rid of it in RR6 https://reactrouter.com/en/main/components/nav-link. It still works as the original so I left it as both the style attribute and classname attribute were already being used.